CONTRIBUTING.md

```diff
@@ -13,4 +13,4 @@ _False_, so users will not get any surprises when upgrading.
 mention of it in the [README](README.md)
 - Make sure changes do not break backwards compatibility, especially with regards to settings.
 - Only changes that stand to benefit a majority of users or use cases are suitable for contributing back to the main repository. For tweaks that are likely specific to your site or likings, try using `CUSTOM_CSS`.
-    - If doing so would require a CSS selector that isn't supported by the theme, create a patch that adds the necessary CSS selector, not the CSS tweak.
+- If doing so would require a CSS selector that isn't supported by the theme, create a patch that adds the necessary CSS selector, not the CSS tweak.
```

Comment:
I think I may have messed this line up, but I don't use github's parser. Happy to revert it if needed.